.github/workflow/doppler-firebase-deploy.yml

```diff
@@ -0,0 +1,40 @@
+# This workflow will install Doppler CLI and Firebase tools, then deploy to Firebase
+
+name: Firebase with Doppler CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    
+    strategy:
+      matrix:
+        node-version: [12.x, 14.x, 16.x]
```

Comment:
Not necessary to test against multiple node versions. Takes 3x longer for the workflow to complete. Change it only test with `node-version: [10.x]` since that's the version that our prod environment uses.

Reply:
If we would like to test against the latest version then we might as well also update the deployments to use the newer version.

In the package.json file there's an engines field that is set to 10. Update that to 14 and it'll shift our prod version to the latest supported by GCP.  Basically we should have the same version be listed in both package.json & our .yml file.